### DISCUSSION TITLE
Furnaces in Desert?

### DISCUSSION THREAD

**Discussion by waynn** [nathan-alden-sr/stardew-valley#38] [Ideas]
I was thinking about this after reading some other guides (in particular TheHaboo's strategy for maxing). So right now you only really get one cycle of smelting when you go to SC because you do it at the beginning of the day then fall asleep in SC. Does it make sense to place the furnaces in the Desert so you can leave at 1:40 am and get one more cycle in? Or do you generally not get enough ore to need to do that (because you don't need quality sprinklers until later)?

**Comment by nathan-alden-sr** [comment]
I have also wondered about this in the past. There are a couple of minor drawbacks:

1. You'd have to leave Skull Cavern about 10-20 in-game minutes minutes before 2:00 am to be able to start the smelting, which means potentially tens of lost Iridium Ore (if you are deep)
2. It will consume an extra Warp Totem: Desert to go pick up the bars the next day if you don't plan on running Skull Cavern (although Totems are cheap to trade for)

I think this strategy could be beneficial, but it's hard to know without trying. I am at Spring 23 in my latest playthrough and struggling to grab enough Copper Ore to craft enough Furnaces to begin with. My normal strategy is to use Warp Totem: Farm items to go back to The Farm and start a batch if I really need one; however, placing the Furnaces right outside the entrance to Skull Cavern would be more efficient because usually the distance to The Farm teleport spot and my Furnances is a lot greater.

I will give this a try. I'm on Spring 23 in my latest playthrough, so this is actually a perfect time to move some Furnaces to The Desert.

I'll let you know how it goes.

**Reply by nathan-alden-sr** [reply]
I made it to Summer 1 without placing Furnaces in The Desert. By the morning of Summer 1--the day you're supposed to ship stuff to buy the remaining Starfruit Seeds--I had 51 Furnaces.
